ui.apps/src/main/content/jcr_root/apps/etoolbox-rollout-manager/clientlibs/rollout-manager-ui/js/console-ui.dialog.js

```diff
@@ -45,26 +48,31 @@
     const PUBLISH_SUCCESS_MSG = Granite.I18n.get('Publishing started');
     const PUBLISH_ERROR_MSG = Granite.I18n.get('Publishing is denied.');
     const ROLLOUT_IN_PROGRESS_LABEL = Granite.I18n.get('Rollout in progress ...');
+    const ROLLOUT_IS_PENDING_LABEL = Granite.I18n.get('Pending (position in queue: {X} of {Y})');
 
     function isLoggerDialog(dialog) {
         return dialog.classList.contains(LOGGER_DIALOG_CLASS);
     }
 
     function loggerDialogFinished(dialog, statusText) {
         if (!isLoggerDialog(dialog)) return;
-        dialog.querySelector('.rollout-processing-label').textContent = statusText;
+        dialog.querySelector('.rollout-processing-status').textContent = statusText;
     }
 
-    function loggerDialogUpdated(dialog) {
-        if (!isLoggerDialog(dialog)) return;
-        dialog.querySelector('.rollout-processing-label').insertAdjacentText('beforeend', ROLLOUT_IN_PROGRESS_LABEL);
+    function updateLoggerDialogStatus(dialog, queue) {
+        const processingLabel = dialog.querySelector('.rollout-processing-label');
+        if (!processingLabel.textContent.trim() || processingLabel.textContent.trim() !== ROLLOUT_IN_PROGRESS_LABEL) {
+            processingLabel.innerText = '';
+            const labelText = queue ? ROLLOUT_IS_PENDING_LABEL.replace('{X}', queue.position).replace('{Y}', queue.total) : ROLLOUT_IN_PROGRESS_LABEL;
+            processingLabel.insertAdjacentText('beforeend', labelText);
+        }
     }
 
     function updateLog(dialog, message) {
         if (message.type !== 'rollout' && message.type !== 'activation') return;
         const itemToUpdate = $(dialog)
             .find('.rollout-log-item')
-            .filter((i, item) => $(item).text() === message.path)
+            .filter((i, item) => $.trim($(item).text()) === message.path)
```

Comment:
Do we really need `JQuery.trim` here?
It is deprecated as far as I know

```suggestion
            .filter((i, item) => ($(item).text() || '').trim() === message.path)
```

Reply:
it can be replaced by native trim method